Card.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Text;
+
+namespace Card_Game
+{
+    public class Card : IComparable<Card>
+    {
+       
+        private Dictionary<CardsAttributes, int> attributes;
+        public int ID { get; private set; }
+        public string Name { get; private set; }
+        public string Description { get; private set; }
+
+        public int this[CardsAttributes key]
+        {
+            get
+            {
+                return attributes[key];
+            }
+        }
+
+        public Card(int ID, string name, int[] attributes, string description)
+        {
```

Comment:
'id' as a parameter instead of ID?

Reply:
There is an example of using "Id":
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalizing-compound-words-and-common-terms

Reply:
Corrected